src/components/CategoryFilter.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+function CategoryFilter(props) {
+  const CATEGORIES = ["ALL", "Action", "Biography", "History", "Horror", "Kids", "Learning", "Sci-Fi"]
+
+  return (
+    <select name="category" onChange={(e) => props.onChange(e)}>
```

Comment:
Naming functions the same as event handlers can get tricky maybe `handleChange`[OPTIONAL]